### ISSUE TITLE
Chapter 7 does not work as specified in book

### ISSUE THREAD

**Issue by bgulian** [fedosejev/react-essentials#5]
Hi, I copied from the book and created my own files via the tutorial.  Twitter photos appear but no buttons or tweets.  I cloned the project and ran Chapter 7 code.  Same result in Firefox and Chrome.  No buttons or tweets.

**Comment by fedosejev** [comment]
Hey @bgulian, have you managed to resolve this?

**Comment by bgulian** [comment]
No.  Do you suggest I repeat my test?  I'd probably be better at debugging
it as I've been working on a major React application for a month and a half
now.

Bob

**Comment by preacher2041** [comment]
HI, I'm having this exact same issue. The Twitter photos are appearing but no buttons or anything else si working.

I've tried to debug it, when I click on on image there is an error saying:
`Uncaught TypeError: Cannot read property '723977503178633216' of undefined`

The line of code that it relates to is:
`var tweet = this.props.tweet[tweetId];`
inside the TweetList.React.js

The line is correct as per the book as far as I can tell so I've hit a bit of an impasse

Any help would be appreciated.

Lee
